spec/system/support_interface/manage_applications_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.feature 'See applications' do
+  scenario 'Provider visits application page' do
+    given_i_am_a_support_user
+    and_there_are_applications_in_the_system
+    and_i_visit_the_support_page
+    then_i_should_see_the_applications
+
+    when_i_click_on_an_application
+    then_i_should_be_on_the_application_view_page
+  end
+
+  def given_i_am_a_support_user
+    # This is stubbed out for now in the controller.
+  end
+
+  def and_there_are_applications_in_the_system
+    create(:application_choice, status: 'application_complete', provider_ucas_code: 'ABC', application_form: create(:application_form, first_name: 'Alice', last_name: 'Wunder'))
```

Comment:
This spec doesn't capture the thing that's unique about the support interface which is that it shows forms, not just choices